packages/spark-extras/components/dictionary/_dictionary.scss

```diff
@@ -0,0 +1,44 @@
+// ==================================================================
+// Dictionary
+// ==================================================================
+
+//
+// Available Variables
+///
+$dictionary-border: 1px solid $gray-medium;
```

Comment:
I like having all the available settings in the settings.scss file since it's easier to make design changes. How do you feel about making a new section under the other component sections in settings for the Dictionary settings?

Reply:
i'll open an issue to talk through this